ProjectManager/ProjectManager/Sources/View/Controllers/AddViewController.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  AddViewController.swift
+//  ProjectManager
+//
+//  Created by Hamo, Wonbi on 2023/01/19.
+//
+
+import UIKit
+
+final class AddViewController: ProjectViewController {
+    private let viewModel: AddViewModel?
+    
+    init(viewModel: AddViewModel) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureNavigation()
+    }
+}
+
+// MARK: Action Method
+extension AddViewController {
+    private func tapCancelButton(_ sender: UIAction) {
+        dismiss(animated: true)
+    }
+    
+    private func tapDoneButton(_ sender: UIAction) {
+        guard let title = textField.text,
+              let description = textView.text
+        else {
+            return
+        }
```

Comment:
그냥 이런식으로 하는게 좀 더 자연스러워 보이네요 저는!
```suggestion
        guard let title = textField.text,
              let description = textView.text else {
              return
        }
```

Reply:
저희가 프로젝트 시작 전에 코드 스타일에 대해서 정리를 한번 하고 시작했었느데요! 그 때 가드문에 여러 조건이 있으면 else 문을 개행하는 것을 규칙으로 삼아서 요렇게 진행하였습니다!

Reply:
알라딘이 제시해주신 부분도 충분히 좋은 방법인것 같습니다!